conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -428,7 +430,10 @@ def create_rebuild_graph(
         requirements = attrs.get("requirements", {})
         host = requirements.get("host", set())
         build = requirements.get("build", set())
-        bh = host or build
+        if include_by_build_only:
```

Comment:
Is this needed, shouldn't we have exported strong exports into the host and run requirements in addition to build?

Reply:
It is missing from host. See this node: https://github.com/regro/cf-graph-countyfair/blob/master/node_attrs/camb.json and this feedstock: https://github.com/conda-forge/camb-feedstock/blob/master/recipe/meta.yaml

I can see if I can fix that.